src/utils/format-srcset/index.js

```diff
@@ -0,0 +1,14 @@
+const formatSrcset = (src, resizerOptions, width, height) =>
+	// TODO : Update comment
```

Comment:
todo comment? is this ready for review?

Reply:
Missed this.  Yeah, need to address this.

Reply:
Thanks, missed this one. Updated.

Reply:
what's your question?

Reply:
@JackHowa : The approach used in `formatSrcset` is slightly different from the code in image here:
https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.1/src/components/image/index.jsx#L36-L48
I think the defensive coding is a good measure. I'm wanting to confirm we're ok with this approach as it will replace the code in the `Image` component.

Reply:
I personally think it's unnecessary logic to have the width and height overriding the option's values: that would be misusing the api based on the docs in Image component. 

But if this is the desire of the team, I don't really think it's a huge deal. Just make sure this optional logic is documented somewhere (what takes precedence over what in what cases). Also that this logic is tested if it's going to stay. 

tldr: Overriding seems like more work and upkeep for an unsupported usecase. 

For example, if a client doesn't want the width and height set on the image component (bad practice for cls), then they can hack us so to speak. They would use the height and width in the resize options. and not include it in the top-level height and width. Also, if a customer puts height and width in the resizer object and in the top-level, they might be confused which we pick.

Reply:
@JackHowa : Thank you very much for your feedback! 👍🏼  I've integrated the srcset-naming util into the `Image` component. Please let me know if you find anything concerning.

Reply:
Image documentation has not been updated to describe which width and height takes precedence, with documentation `Just make sure this optional logic is documented somewhere (what takes precedence over what in what cases).`

Reply:
height and width are not set on the source element if the height and width of the srcset are generated from the resizedOptions

Reply:
I addressed this in commit